ChangeLog.d/8340.txt

```diff
@@ -1,4 +1,4 @@
 Features
-   * mbedtls_md_psa_alg_from_type() and mbedtls_md_type_from_psa_alg() helper
-     functions were added to convert from mbedtls_md_type_t to psa_algorithm_t
-     and viceversa.
+   * Add functions mbedtls_md_psa_alg_from_type() and
+     mbedtls_md_type_from_psa_alg() to convert between mbedtls_md_type_t and
+     psa_algorithm_t and vice versa.
```

Comment:
I think "between ... and ..." already means it goes both way, so the "and vice versa" is superfluous